src/components/ReactIcon/ReactIcon.tsx

```diff
@@ -0,0 +1,13 @@
+import { IconContext } from 'react-icons';
+import { PropsWithChildren } from 'react';
+
+interface ReactIconProps extends PropsWithChildren {
+  value: object;
+}
+export const ReactIcon = ({ value, children }: ReactIconProps) => {
```

Comment:
Just call it Icon. Prefixes like React or Component are useless. Names should be short but concise and descriptive at the same time. Here is a good reading for you about them https://github.com/ryanmcdermott/clean-code-javascript#variables